File: baekjoon/shim/Java/단계별로_풀어보기/문자열/n_1152.java
package Study_2022.baekjoon.shim.Java.단계별로_풀어보기.문자열;

import java.util.*;

public class n_1152 {
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        String st[]=sc.nextLine().split(" ");
        int len=st.length;
        for(int i=0; i<st.length; i++){
            if(st[i]=="")
                len--;
        }
        System.out.println(len);
    }
}

File: baekjoon/shim/Python/단계별로_풀어보기/문자열/n_1152.py
l=input().split(" ")
cnt=len(l)
for i in l:
    if i=='':
        cnt-=1
print(cnt)

# 다른 풀이
# print(len(input().split()))

# split()과 split(" ")의 다른점
# split() 공백이 1개이건 여러개이건 1개로 보고 처리한다
# split(" ") 공백을 각각의 공백으로 보고 처리한다.